Source/EasyNetQ.Hosepipe.Tests/packages.config

```diff
@@ -2,5 +2,5 @@
 <packages>
   <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net40" />
   <package id="NUnit" version="2.6.3" targetFramework="net40" />
-  <package id="RabbitMQ.Client" version="3.5.4" targetFramework="net40" />
+  <package id="RabbitMQ.Client" version="3.5.5" targetFramework="net4" />
```

Comment:
@zidad just saw your comment now, you can leave it, just hope to be retro-compatible, let's see if someone complains, build server did well, so probably there will be no problem.